bls/src/types/public_key.rs

```diff
@@ -72,6 +75,24 @@ impl PublicKey {
         let public_key = self.public_key.mul_bigint([x as u64]);
         PublicKey { public_key }
     }
+
+    pub fn trusted_serialize(&self) -> [u8; PublicKey::TRUSTED_SERIALIZATION_SIZE] {
```

Comment:
I would prefer to see a rustdoc, explaining what makes this trusted and how it can fail.

Reply:
Done. Cannot fail.